string garage_id
string linked_rover_id
string state
bool lights_on
string health
string health_details
bool rover_docked